debian/config

```diff
@@ -2,6 +2,15 @@
 
 set -e
 
+# There are two questions open:
+
+# wartool -m  is inconsistently documented. It can be easily added here, but is it still part of wartool?
+
+# how should the Expansion Pack (Beyond the Dark Portal) be handled? I've
```

Comment:
Sorry, was working down my mailbox, so commented on the closed PR.

Anyway, the expansion CD can be extracted on its own and the game will work, but to get both the ToD and BtDP campaigns you need to extract first from the base game and then from the expansion CD. Alternatively, I'm now building the wargus debian packages with StormLib, so the Battle.net Edition should also work and then you get both campaigns from one CD.